utils/build.ps1

```diff
@@ -334,7 +334,7 @@ function Get-InstallDir($Arch) {
   return "$ImageRoot\$ProgramFilesName\Swift"
 }
 
-function Get-HostSwiftSDK() {
+function Get-BuildSwiftSDK() {
```

Comment:
Wait, this is the host ... I was wrong. I'm confused, why are we not passing in the `Arch` for the host arch sdk? Could we not cross-compile to more than one arch?

Reply:
because there's only one host arch?

Reply:
Ah right, this is the host SDK for the toolchain, not an arbitrary host SDK. I think that generalising this to `Get-SwiftSDK($Arch)` might be nice as it currently makes me think that it is getting the Swift SDK for a particular host (which is true, it's just that the particular host happens to be the host that we are building the toolchain for). But, lets leave that for a follow up, I'd like to get this in.